lib/driverProviders/driverProvider.js

```diff
@@ -5,6 +5,7 @@
  */
 
 var webdriver = require('selenium-webdriver'),
+    _ = require('lodash'),
```

Comment:
Hey, 

there's only a little thing to do anymore, to make the Pull request succeed. Let's do it! :) 

The above line thus should only have:
var webdriver = require('selenium-webdriver') 

and nothing else. The part mentioning _ (loading lodash module) should be removed.